app/views/posts/index.html.erb

```diff
@@ -1,3 +1,52 @@
-<h1>Here is a page for showing list for post for a given user </h1>
+<div class="user_con">
 
-<p>This page is under construction</p>
+    <div class="user_data">
+
+      <h2><%= @user.name%></h2>
+
+      <p>Number of posts: <%= @user.posts_counter %></p>
+
+    </div>
+
+  </div>
+
+
+
+
+<% @posts.each do |post| %>
+
+  <div class="user_con">
```

Comment:
- [ ] Posts index page is throwing an error, kindly check it and make sure that 5 recent comments are being displayed:
<img width="1009" alt="Screenshot 2022-05-03 at 12 49 25" src="https://user-images.githubusercontent.com/77038610/166426769-e0c7a26a-7424-4b1f-b225-05450e728b39.png">